models/public/ssh-mxnet/model.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  SSH: Single Stage Headless Face Detector. More details provided in the repository
+  <https://github.com/deepinsight/mxnet-SSH> and paper <https://arxiv.org/abs/1708.03979>
+task_type: detection
+files:
+  - name: ssh-model-final.zip
+    size: 73316283
+    sha256: 1ac43c830fe8f5de069a35101cc38b03751c8502e9c62e9a945424a05e25dc6c
+    source: https://www.dropbox.com/s/06zmchmq4iwj6rt/ssh-model-final.zip?dl=1
```

Comment:
Where does this link come from? I don't see any mentions of it in the repository.

Reply:
Link to the pre-trained model comes from https://github.com/deepinsight/insightface/wiki/Model-Zoo#1-face-detection-models

Reply:
OK. Weird that they would publish it like that.